doc/release/1.9.0-notes.rst

```diff
@@ -70,60 +145,806 @@ New features
+* `#15762 <https://github.com/scipy/scipy/issues/15762>`__: DEP: remove stats.median_absolute_deviation
+* `#15773 <https://github.com/scipy/scipy/issues/15773>`__: BUG: iirfilter allows Wn[1] < Wn[0] for band-pass and band-stop...
+* `#15780 <https://github.com/scipy/scipy/issues/15780>`__: BUG: CI on Azure broken with PyTest 7.1
+* `#15854 <https://github.com/scipy/scipy/issues/15854>`__: CI: Windows Meson job failing sometimes on OpenBLAS binary download
+* `#15866 <https://github.com/scipy/scipy/issues/15866>`__: BUG/CI: Wrong python version used for tests labeled "Linux Tests...
+* `#15899 <https://github.com/scipy/scipy/issues/15899>`__: BUG: _calc_uniform_order_statistic_medians documentation example...
+* `#15927 <https://github.com/scipy/scipy/issues/15927>`__: BUG: Inconsistent handling of INF and NAN in signal.convolve
+* `#15931 <https://github.com/scipy/scipy/issues/15931>`__: BUG: scipy/io/arff/tests/test_arffread.py::TestNoData::test_nodata...
+* `#15960 <https://github.com/scipy/scipy/issues/15960>`__: BUG: Documentation Error in scipy.signal.lfilter
+* `#15961 <https://github.com/scipy/scipy/issues/15961>`__: BUG: scipy.stats.beta and bernoulli fails with float32 inputs
+* `#15962 <https://github.com/scipy/scipy/issues/15962>`__: Race condition in macOS Meson build between \`_matfuncs_expm\`...
+* `#15987 <https://github.com/scipy/scipy/issues/15987>`__: CI: \`np.matrix\` deprecation warning
+* `#16011 <https://github.com/scipy/scipy/issues/16011>`__: BUG: typo in documentation for scipy.optimize.basinhopping
+* `#16020 <https://github.com/scipy/scipy/issues/16020>`__: BUG: dev.py FileNotFoundError
+* `#16027 <https://github.com/scipy/scipy/issues/16027>`__: jc should be (n-1)/2
+* `#16031 <https://github.com/scipy/scipy/issues/16031>`__: BUG: scipy.sparse.linalg.norm does not work on sparse arrays
+* `#16036 <https://github.com/scipy/scipy/issues/16036>`__: Missing \`f\` prefix on f-strings
+* `#16054 <https://github.com/scipy/scipy/issues/16054>`__: Bug: Meson build with dev.py fails to detect SciPy with debian...
+* `#16065 <https://github.com/scipy/scipy/issues/16065>`__: BUG: Gitpod build with \`python runtests.py\` fails; move to...
+* `#16074 <https://github.com/scipy/scipy/issues/16074>`__: BUG: refguide check fails with \`numpydoc==1.3\`
+* `#16081 <https://github.com/scipy/scipy/issues/16081>`__: CI, MAINT: minor refguide failure with stats.describe
+* `#16121 <https://github.com/scipy/scipy/issues/16121>`__: DOC: scipy.interpolate.RegularGridInterpolator and interpn works...
+* `#16162 <https://github.com/scipy/scipy/issues/16162>`__: BUG: curve_fit gives wrong results with Pandas float32
+* `#16171 <https://github.com/scipy/scipy/issues/16171>`__: BUG: scipy.stats.multivariate_hypergeom.rvs raises ValueError...
 
 ***********************
 Pull requests for 1.9.0
 ***********************
 
+* `#354 <https://github.com/scipy/scipy/pull/354>`__: Faster expm
```

Comment:
```suggestion
```
This wasn't actually merged; it was superseded. Perhaps the code that generates this list could use a little tweak? Not too important, though.